ManalHamdi/assignment2/Assignments2_Q1.cc

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <queue>
+#include <vector>
+
+struct Node {
+	int data;
+	Node* parent;
+	Node* left_child;
+	Node* right_child; 
+};
+class Tree {
+public:
+	Node* root;
+	Tree() {
```

Comment:
The same comments about destructors, static functions and moving `CreateRoot` to `Tree(int data)` :)